### DISCUSSION TITLE
Does DIA-NN perform any kind of spectrum pre-processing to raw files before the analysis?

### DISCUSSION THREAD

**Discussion by ec-ho-ra-mos** [vdemichev/DiaNN#586] [General]
Hello Professor Demichev,

I am wondering whether DIA-NN performs any kind of pre-processing to raw files (i.e. peak scaling, peak normalization, signal-to-noise filtering, etc.). According to [Fig. 1 | DIA-NN workflow and its performance on conventional and short chromatographic gradients](https://www.nature.com/articles/s41592-019-0638-x) DIA-NN should not be performing any of these and there is also no mention of traditional spectrum preprocessing in the paper, but I wanted to ask to be sure.

**Comment by vdemichev** [comment]
Yes, there's mass calibration, the details of that are unpublished.

**Reply by ec-ho-ra-mos** [reply]
Thank you very much for answering my question.
